### DISCUSSION TITLE
Installing Homebrew to Mac M1 mini

### DISCUSSION THREAD

**Discussion by darrenkoong** [Homebrew/discussions#1800] [Getting started]
I am new to homebrew and wanted to install to my mac m1 mini. I am using the link provided in brew.sh to install. Below is the link.
/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"

After installing, the word (base) appeared at the front my mac mini name in terminal. Is this correct? How do I remove the word (base) on my terminal?

**Comment by alxrey** [comment]
Did you only install Homebrew ?
It seems it's linked to conda environment (anaconda or minconda) not to homebrew.

**Reply by darrenkoong** [reply]
yes, you are right. I was running anaconda and have not deactivate it. Thanks.

**Reply by alxrey** [reply]
You're welcome. 👍
